### DISCUSSION TITLE
wm.bar.font

### DISCUSSION THREAD

**Discussion by b4h4w** [regolith-linux/regolith-desktop#1058] [Q&A] [answered]
Hi guys! I'm trying to change the bar font through Xresources using wm.bar.font, but it only applies to the workspace name.  Is there a specific way to apply this on date/time, battery, etc?

**Comment by kgilmer** [comment, marked as answer]
Hi @b4h4w , are you looking to change the text, the icon, or both?  You might try [setting `i3xrocks.value.font`..](https://github.com/regolith-linux/regolith-look-default/blob/main/usr/share/regolith-look/default/i3xrocks#L37)

**Reply by b4h4w** [reply]
Just the text and it works! This is exactly what I'm looking for. Thanks, @kgilmer
